lib/presenter/formatter_csv_classification.rb

```diff
@@ -58,6 +58,14 @@ def annotations
       def expert
         classification.expert_classifier
       end
+
+      def obfuscate_value(value)
+        if obfuscate
+          "#{value}#{salt}".hash
```

Comment:
As far I can tell Ruby uses Murmur Hash for Object#hash. I think if we actually want to make sure this value is obfuscated we should at least use some sort of cryptographic hash like SHA-1 or -2.